.cursor/skills/waiting-for-build/scripts/query.py

```
@@ -98,49 +109,113 @@ def jira_get(path: str, headers: dict, base: str) -> dict:
+    """
+    try:
+        r = requests.get(
+            f"{base}/rest/agile/1.0/board/{board_id}/sprint?state=active",
+            headers=headers, timeout=20)
+        if r.ok:
+            sprints = r.json().get("values", [])
+            if sprints:
+                return sprints[0]
+        # 401 / 403 on SSO instances: fall through to JQL fallback
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)